### PR TITLE
node/http: pprof endpoint fix

### PR DESCRIPTION
**Pull request by Exca-DK**

### CODE REVIEW

File: node/http/server.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"time"
 
 	"github.com/NethermindEth/juno/db"
-	"github.com/NethermindEth/juno/grpc"
+	junogrpc "github.com/NethermindEth/juno/grpc"
```
Comment (Exca-DK) [inline node/http/server.go:12]: The rename is due to go default formatter behavior.

Comment (joshklop) [reply node/http/server.go:12]: I believe this is due to VSCode defaults. Are you able to change the VSCode formatter to `gofumpt` for this project?

**Comment by joshklop** [comment]
Interesting that the test did not catch this issue...

File: node/http/server.go (hunk 1 above)
Comment (omerfirmak) [reply node/http/server.go:12]: Can we revert this change? Since it is unrelated.

File: node/http/server.go [hunk 2]
```diff
@@ -62,7 +62,7 @@ func New(listener net.Listener, metrics bool, database db.DB, version string,
 	mux.Handle("/grpc", grpcHandler)
 	if pprofEnable {
 		// Taken from https://artem.krylysov.com/blog/2017/03/13/profiling-and-optimizing-go-web-applications/
```
Comment (joshklop) [inline node/http/server.go:64]: Let's remove this comment now that we're changing the code. Almost identical lines are in the stdlib anyway.

File: node/http/server.go (hunk 1 above)
Comment (Exca-DK) [reply node/http/server.go]: Sure.

**Comment by Exca-DK** [comment]
> Interesting that the test did not catch this issue...
> 
> EDIT: just realized this is a change to align behavior with stdlib, not a bug fix.

It is in fact a bugfix. Without the slash it is impossible to access /debug/pprof/heap or /debug/pprof/goroutine

**Review by joshklop — APPROVED** [review]
LGTM.

**Pull request merged**